src/index.js

```diff
@@ -415,7 +415,7 @@ Multiaddr.protocols = protocols
  */
 Multiaddr.isMultiaddr = function isMultiaddr (addr) {
   if (addr.constructor && addr.constructor.name) {
-    return addr.constructor.name === 'Multiaddr'
```

Comment:
Wait, this will fail terribly when we have multiple version of Multiaddr flying around